sf-stream/src/tests/accounts_test.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Aptos
```

Comment:
Are these tests relevant?

Reply:
none of these tests are relevant yet; it's in the TODO for the PR :-)